src/events.js

```diff
@@ -96,12 +94,14 @@ module.exports = function(ctx) {
     else if (event.keyCode === 51) {
       ctx.api.changeMode('draw_polygon');
     }
+    ctx.ui.fireClassUpdate();
   };
 
   events.keyup = function(event) {
     if (isKeyModeValid(event.keyCode)) {
       currentMode.keyup(event);
     }
+    ctx.ui.fireClassUpdate();
```

Comment:
ok, I lied in the first comment. We need to do this for each event type in events.js and we should do it anywhere else in Draw.